cairo/src/lib.rs

```diff
@@ -86,6 +86,21 @@ macro_rules! gvalue_impl {
             }
         }
 
+        impl From<$name> for glib::Value {
+            fn from(v: $name) -> Self {
+                unsafe {
+                    let mut value =
+                        glib::Value::from_type(<$name as glib::StaticType>::static_type());
+                    glib::gobject_ffi::g_value_take_boxed(
+                        value.to_glib_none_mut().0,
+                        glib::translate::ToGlibPtr::to_glib_none(&&v).0 as *mut _,
+                    );
+                    let _v = std::mem::ManuallyDrop::new(v);
```

Comment:
Why not `into_glib_ptr()`?

Reply:
There was not any implementations of it, I wrote some